compiler/control/OMROptions.cpp

```diff
@@ -732,6 +733,7 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
    {"enableSpecializedEpilogues",         "O\tapply restore non-volatiles selectively",         SET_OPTION_BIT(TR_EnableSpecializedEpilogues), "F"},
    {"enableTailCallOpt",                  "R\tenable tall call optimization in peephole", SET_OPTION_BIT(TR_EnableTailCallOpt), "F"},
    {"enableThisLiveRangeExtension",       "R\tenable this live range extesion to the end of the method", SET_OPTION_BIT(TR_EnableThisLiveRangeExtension), "F"},
+   {"enableTrampolines",                  "O\tenable generation of trampolines for method dispatch", SET_OPTION_BIT(TR_EnableTrampolines), "F"},
```

Comment:
This option only enables zOS64 trampolines as I believe the feature will not be enabled by default yet. Perhaps renaming it to enablezOSTrampolines would make it clearer.